src/sennet_translator.py

```diff
@@ -307,6 +329,10 @@ def translate_all(self):
 
             except Exception as e:
                 logger.error(e)
+            finally:
+                if progress_writer is not None:
+                    progress_writer.reset()
```

Comment:
Once the full reindex completes the `percent_complete` is reset to 0, we should hold it at whatever percent it is (ideally 100) until we issue another `reindex-all` call. We should, however, still reset the `is_indexing` property.
![image](https://github.com/user-attachments/assets/2e0270b0-769b-4c77-8da3-11b85a07eb59)

Reply:
When I simply replaced this with: `progress_writer.is_indexing = False` its stuck at 72% locally
<img width="299" alt="image" src="https://github.com/user-attachments/assets/6b29902f-f837-4936-8d94-d7086580abbf" />
Maybe we just set it to 100 manually inside the `finally`

Reply:
@maxsibilla I just pushed some changes that resolve those issues